vlemish/BinaryTree.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace DevChallengesSolution;
+
+public class BinaryTree
+{
+    // We need to traverse to very left and very right nodes of the tree until we reach the leaf
+    // in every node we need to swap position of each sub tree. Left node will become right and vice versa.
+    // To traverse to very left and very right nodes we can use postorder traversal.
+    // In this case, at first we'll go to left and
+    // swap nodes and then we'll go to right and do the same thing.
+    // At the end we'll swap nodes in the root and this will solve the problem.
+    public TreeNode InvertTree(TreeNode root)
```

Comment:
Very elegant recursive solution even selecting the most suitable traversal. Nice one.